.travis.yml

```diff
@@ -1,16 +1,12 @@
 language: python
 python:
  - 2.7
+ - 3.5
 sudo: false
-env:
- - TOX_ENV=py27-unittest
- - TOX_ENV=py27-integration
- - TOX_ENV=lint
```

Comment:
This should continue to run linking and the other 2.7 tests -- you should add 3.5 tests to this.